python/activator/middleware_interface.py

```diff
@@ -75,6 +75,12 @@ class MiddlewareInterface:
         appropriate for use in the Google Cloud environment; typically only
         change this when running local tests.
     """
+    _COLL_TEMPLATE = "templates"
```

Comment:
Can we call these `_COLLECTION_TEMPLATE` instead, if we're going with an all-caps name?